core/src/components/range/range.tsx

```diff
@@ -191,6 +191,16 @@ export class Range implements ComponentInterface {
    */
   @Event() ionBlur!: EventEmitter<void>;
 
+  /**
+   * Emitted when the user starts moving the range knob.
```

Comment:
~ Should we call out this fires with both keyboard input and dragging the knob?

Reply:
Good thinking, will update :eyes: